marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/updateMergeOptionsLib.sjs

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+function updateMergeOptions(opt)
```

Comment:
Can you add unit tests for these functions?

Reply:
Oops, misplaced that file and found it in an auto-stash. Added in latest push.